doc/index.rst

```diff
@@ -185,3 +184,4 @@ Strawberry Fields is **free** and **open source**, released under the Apache Lic
    code/sf_program
    code/sf_program_utils
    code/sf_parameters
+   code/sf_api
```

Comment:
So moving this line for where the deleted line is

Reply:
Thanks for pointing that out, done :+1: